docs/source/building-supergraphs/subgraphs-apollo-server.mdx

```diff
@@ -137,24 +164,27 @@ const resolvers = {
 
 Finally, we use the `buildSubgraphSchema` function from the `@apollo/subgraph` package to augment our schema definition with federation support. We provide the result of this function to the `ApolloServer` constructor:
 
-```js title="index.js"
+<MultiCodeBlock>
+
+```ts title="index.ts"
 const server = new ApolloServer({
   schema: buildSubgraphSchema({ typeDefs, resolvers })
 });
-
-server.listen(4001).then(({ url }) => {
```

Comment:
You're not replacing this, which is a bit confusing because the next section says it's just combining the examples and does have a replacement.

Reply:
Oh great catch, thank you! Updated to:
```ts title="index.ts"
const server = new ApolloServer({
  schema: buildSubgraphSchema({ typeDefs, resolvers })
});

// Note the top level await!
const { url } = await startStandaloneServer(server);
console.log(`🚀  Server ready at ${url}`);
```